### PR TITLE
15 improve developer experience by restructuring files

### LINKED ISSUE #15: Improve Developer Experience by Restructuring Files
## Description

The current file structure allows fexibility for resources to be environment specific, however it is at the cost of a bad developer experience. The current structure requires imports of new resources developed in the console to first be imported into an environment folder and followed by a number of manual steps for effective promotion. 

Our developer lifecycle expects users to develop within the console, so supporting a better export/promotion experience is the priority.

### PR DESCRIPTION
**Pull request by samir-gandhi**

### CODE REVIEW

**Comment by samir-gandhi** [comment]
includes upgrade to v1.0.0
closes #9

File: secretstemplate [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
-export TF_VAR_pingone_davinci_environment_id=''
-# group_id for davinci environment terraform user accounts
+export TF_VAR_pingone_davinci_admin_username=''
+export TF_VAR_pingone_davinci_admin_password=''
+export TF_VAR_pingone_davinci_admin_environment_id=''
+# one of: "NorthAmerica | Canada | Asia | Europe"
+export TF_VAR_pingone_davinci_admin_region=''
+export TF_VAR_pingone_davinci_user_group_id=''
 export TF_VAR_pingone_davinci_terraform_group_id=''
+export TF_VAR_pingone_environment_type=""
```
Comment (PingDavidR) [inline secretstemplate]: should we give the "one of: comment to guide the user here?  I think it'll be either SANDBOX or PRODUCTION, right?

File: secretstemplate [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+
+
+## from CICD repo
 # aws credentials and information for dev environment
 export AWS_ACCESS_KEY_ID=''
 export AWS_SECRET_ACCESS_KEY=''
-# bucket name for storing state - for example, 'ping-terraform-demo-mycompany'
-export TF_VAR_tf_state_bucket=''
+# bucket name for storing state - for example, 'ping-terraform-demo-mycompany'a
+export TF_VAR_tf_state_bucket='ping-terraform-demo'
```
Comment (PingDavidR) [inline secretstemplate]: remove your bucket name from this file, use env var

**Review by PingDavidR — APPROVED** [review]
approving pending couple of comments

File: secretstemplate (hunk 2 above)
Comment (samir-gandhi) [reply secretstemplate]: corrected

File: secretstemplate (hunk 1 above)
Comment (samir-gandhi) [reply secretstemplate]: done

**Pull request merged**